package.json

```diff
@@ -8,13 +8,16 @@
     "install-react:0.14": "yarn run install-react -- react@^0.14.9 react-dom@^0.14.9",
     "install-react:15": "yarn run install-react -- react@^15.5.4 react-dom@^15.5.4",
     "lint": "xo --quiet",
+    "start": "./packages/react-cosmos-webpack/bin/cosmos.js --config examples/local-state/cosmos.config.js",
     "build": "node ./scripts/build.js",
     "postinstall": "yarn run bootstrap && yarn run build",
     "publish-lerna": "yarn run test && yarn run build && lerna publish",
     "test-karma": "karma start test/karma.conf.js --single-run",
     "test-jest": "jest --coverage --config test/jest.config.json",
     "test": "yarn run lint && yarn run test-karma && yarn run test-jest",
-    "coveralls": "lcov-result-merger 'test/coverage/*/lcov.info' | node_modules/coveralls/bin/coveralls.js"
+    "coveralls": "lcov-result-merger 'test/coverage/*/lcov.info' | node_modules/coveralls/bin/coveralls.js",
+    "test-cypress": "cypress run",
```

Comment:
Will rename. I see them as smoke tests as they don't (and can't, because Cypress doesn't work with iframes 😞 ) cover all functionality.